.github/stale.yml

```diff
@@ -1,24 +1,41 @@
 # https://github.com/marketplace/stale
+# https://github.com/probot/stale
+
+issues:
+    # Number of days of inactivity before an issue becomes stale
+    daysUntilStale: 30
+    # Number of days of inactivity before a stale issue is closed
+    daysUntilClose: 7
+    # Issues with these labels will never be considered stale
+    exemptLabels:
+      - Important
+      - Priority
+    # Comment to post when marking an issue as stale. Set to `false` to disable
+    markComment: >
+      This issue has been automatically marked as stale because it has not had recent activity.
+      It will be closed if no further activity occurs. Thank you for your contributions.
+    # Comment to post when closing a stale issue. Set to `false` to disable
+    closeComment: false
+
+pulls:
+    # Number of days of inactivity before an pulls becomes stale
+    daysUntilStale: 14
+    # Number of days of inactivity before a stale pull is closed
+    daysUntilClose: 5
+    # Label to use when marking an issue as stale
+    staleLabel: "won't fix"
+    # Comment to post when marking an issue as stale. Set to `false` to disable
+    markComment: >
+      This pull request has been automatically marked as stale because it has not had recent activity.
+      It will be closed if no further activity occurs. If you need further help see our docs:
+      https://pytorch-lightning.readthedocs.io/en/latest/CONTRIBUTING.html#pull-request
+      or ask assistance of core contributor here or on Slack.
+      Thank you for your contributions.
```

Comment:
```suggestion
      This pull request has been automatically marked as stale because it has not had recent activity.
      It will be closed in 7 days if no further activity occurs. If you need further help see our docs:
      https://pytorch-lightning.readthedocs.io/en/latest/CONTRIBUTING.html#pull-request
      or ask assistance of core contributor here or on Slack.
      Thank you for your contributions.
```